spec/fixtures/cassettes/BookingSync_API_Client_Bookings/_cancel_booking/cancels_given_booking_and_passes_options_along_as_bookings_payload.yml

```diff
@@ -68,6 +68,6 @@ http_interactions:
     body:
       encoding: UTF-8
       string: ''
-    http_version: 
+    http_version:
   recorded_at: Mon, 12 Dec 2016 12:13:17 GMT
```

Comment:
I would rather record it myself than let it be merged like that :P  Shoudl I handle it then?